modules/ossm-excluding-namespaces-from-cluster-wide-mesh-cli.adoc

```diff
@@ -5,8 +5,8 @@
 [id="ossm-excluding-namespaces-from-cluster-wide-mesh-cli_{context}"]
 = Including and excluding namespaces from a cluster-wide mesh by using the CLI
 
-By default, the {SMProductName} Operator uses discovery selectors to identify the namespaces that make up the mesh. Namespaces that do not contain the label defined in the `ServiceMeshMemberRoll` resource are not matched by the discovery selector and are excluded from the mesh.  
- 
+In a cluster-wide mesh, you add discovery selectors to the `ServiceMeshControlPlane` resource. Discovery selectors define which namespaces the control plane can discover. The control plane ignores any namespace that does not match one of the discovery selectors, which excludes the namespace from the mesh.
```

Comment:
> In a cluster-wide mesh, you add discovery selectors...

Adding discovery selectors is optional, but I think this makes it sound like a requirement. I also think that adding something about limiting scope/visibility could give readers a more immediate understanding of what discovery selectors do. How about something like the following?

> In a cluster-wide mesh, discovery selectors can be added to the `ServiceMeshControlPlane` resource. Discovery selectors limit the scope of the control plane to a subset of the namespaces in the cluster. The control plane ignores any namespace that does not match one of the discovery selectors, which excludes the namespace from the mesh.

Reply:
done